### DISCUSSION TITLE
Customizable NyForm Picker

### DISCUSSION THREAD

**Discussion by alecoud** [nylo-core/nylo#190] [Ideas] [answered]
Hello @agordn52, will you conisder add customization support for the NyForm Picker? Which includes the picker field style, and the modal bottom sheet style for the options, in our project we need to customize the showModalBottomSheet widget builder, backgroundColor, barrierColor, useRootNavigator, routeSettings.

**Comment by agordn52** [comment]
Hey @alecoud,

Yeah, I think these changes would be amazing. 
I'll see what I can do, keep you updated.

**Reply by alecoud** [reply]
That's great, I am looking forward to the next update.

**Reply by agordn52** [reply, marked as answer]
Hey @alecoud,

You'll now be able to use `BottomModalSheetStyle` like this

``` dart
Field.picker(
          "Keywords",
          options: ["One", "Two", "Three"],
          style: "compact",
          bottomModalSheetStyle: BottomModalSheetStyle(
            backgroundColor: NyColor(light: Colors.red, dark: Colors.blue),
            titleStyle: NyTextStyle(fontWeight: FontWeight.bold, backgroundColor: NyColor(light: Colors.blueAccent), color: NyColor(light: Colors.green, dark: Colors.yellow)),
            itemStyle: NyTextStyle(color: NyColor(light: Colors.purple, dark: Colors.yellow)),
            clearButtonStyle: NyTextStyle(color: NyColor(light: Colors.blue, dark: Colors.yellow)),
            barrierColor: NyColor(light: Colors.amberAccent),
            routeSettings: ...,
            useRootNavigator: ...
          ),
        ),
```

Added in `nylo_framework: ^6.4.0` 🎉

**Reply by alecoud** [reply]
Hi @agordn52, many thanks for the update, I will have a try.
